src/IceRpc.Templates/Templates/IceRpc-Protobuf-Client/proto/greeter.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) ZeroC, Inc.
+
+syntax = "proto3";
+
+package visitor_center;
+option csharp_namespace = "IceRpc_Protobuf_Client";
+
+// The protobuf equivalent of Slice interface Greeter in the Greeter example.
```

Comment:
This is not a good description. Protobuf examples / templates should not reference Slice.